tools/__tasks__/compile/javascript/webpack-dcr.dev.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-unassigned-import
+require('any-observable/register/rxjs-all');
+
+const Observable = require('any-observable');
+
+const webpack = require('webpack');
+const chalk = require('chalk');
+
+const config = require('../../../../webpack.config.dcr.dev.js');
+
+module.exports = {
+    description: 'Create Webpack commercial.js bundle for dotcom-rendering',
+    task: () =>
```

Comment:
The `task` function in `webpack-dcr.dev.js` and `webpack-dcr.js` is pretty much identical, this repition is present across all our build tasks, we considered refactoring it out but would rather not update all our existing build tasks in this PR.